changes/S821-027.yaml

```diff
@@ -0,0 +1,8 @@
+type: api-change
+title: Rename ``BasicDecl.p_is_unit_root`` into ``p_is_compilation_unit_root``
+
+description: |
+    To be closer to common Ada terminology, and disambiguate between
```

Comment:
> Rename the p_is_unit_root property on BasicDecl into p_is_compilation_unit_root. This makes it closer to […]

I don't see it as more helpful. What do you think is more helpful ? that you phrase it as "`p_is_unit_root` property on `BasicDecl`" rather than "`BasicDecl.p_is_unit_root`" ? I think the dot notation is the only element in all of that that will be understandable by everyone, and it is the only thing you change.

"property" is langkit jargon too, and the naming convention is the python one, but we have to choose one for those changelogs I guess. I personally don't find your version more helpful. I also don't see the interest of repeating the description if the title already contains it. That won't make it more readable or more helpful, and repeating the same thing twice raises the risk of making a mistake.

So I would rather change the proposal to be more precise and say that the description doesn't have to be repeated.

Reply:
Ok